Sources/BrowserServicesKit/Autofill/AutofillUserScript+SourceProvider.swift

```diff
@@ -58,7 +58,39 @@ public class DefaultAutofillSourceProvider: AutofillUserScriptSourceProvider {
         replacements["// INJECT contentScope HERE"] = "contentScope = " + privacyConfigJson + ";"
         replacements["// INJECT userUnprotectedDomains HERE"] = "userUnprotectedDomains = " + userUnprotectedDomainsString + ";"
         replacements["// INJECT userPreferences HERE"] = "userPreferences = " + jsonPropertiesString + ";"
+        // TODO: use dynamic values
+        let availableInputTypes = """
```

Comment:
This also conforms to https://github.com/duckduckgo/duckduckgo-autofill/blob/7f8e8cbe54ee581f33cc82eca8c9f96135d8ef0e/src/deviceApiCalls/schemas/availableInputTypes.json.